iib/workers/tasks/build.py

```diff
@@ -703,3 +707,68 @@ def handle_regenerate_bundle_request(from_bundle_image, organization, request_id
         'state_reason': 'The request completed successfully',
     }
     update_request(request_id, payload, exc_msg='Failed setting the bundle image on the request')
+
+
+def _copy_files_from_image(image, src_path, dest_path):
+    """
+    Copy a file from the container image into the given destination path.
+
+    The file may be a directory.
+
+    :param str image: the pull specification of the container image.
+    :param str src_path: the full path within the container image to copy from.
+    :param str dest_path: the full path on the local host to copy into.
+    """
+    # One way to copy a file from the image is to create a container from its filesystem
+    # so the contents can be read. To create a container, podman always requires that a
+    # command for the container is set. In this method, however, the command is not needed
+    # because the container is never started, only created. Use a dummy command to satisfy
+    # podman.
+    container_command = 'unused'
+    container_id = run_cmd(
+        ['podman', 'create', image, container_command],
+        exc_msg=f'Failed to create a container for {image}',
+    ).strip()
+    try:
+        run_cmd(
+            ['podman', 'cp', f'{container_id}:{src_path}', dest_path],
+            exc_msg=f'Failed to copy the contents of {container_id}:{src_path} into {dest_path}',
+        )
+    finally:
+        try:
+            run_cmd(
+                ['podman', 'rm', container_id],
+                exc_msg=f'Failed to remove the container {container_id} for image {image}',
+            )
+        except IIBError as e:
+            # Failure to remove the temporary container shouldn't cause the IIB request to fail.
+            log.warning(e)
```

Comment:
Optional: Perhaps `log.exception` would be better to get the traceback

Reply:
Good idea. I changed it to `log.exception`. I left `e` as a parameter because that already has the error message we care about.